src/scripts/main.js

```diff
@@ -1,3 +1,261 @@
 'use strict';
 
-// write code here
+const pageBody = document.querySelector('body');
+const tableHeader = document.querySelector('thead');
+const tableHeaderCells = tableHeader.querySelector('tr');
+const tableBody = document.querySelector('tbody');
+let isSecondClick = false;
+const toNumber = (string) => {
+  if (typeof string === 'string' && string.length !== 0) {
+    const clearNum = string.replace(/[$, ]/g, '');
+
+    return parseInt(clearNum, 10);
+  }
+};
+let clickedSortTable;
+
+tableHeaderCells.addEventListener('click', (e) => {
+  const headerChildren = [...tableHeaderCells.children];
+  const columnName = e.target.closest('th');
+
+  if (columnName !== null) {
+    const sortedColumn = headerChildren.indexOf(columnName);
+    const columnsToSort = [...tableBody.children];
+
+    if (clickedSortTable === columnName) {
+      isSecondClick = !isSecondClick;
+    } else {
+      isSecondClick = false;
+    }
+
+    clickedSortTable = columnName;
+
+    const columnsSorted = columnsToSort.sort((a, b) => {
+      const elemA = a.children[sortedColumn].innerHTML;
+      const elemB = b.children[sortedColumn].innerHTML;
+
+      const aToNum = toNumber(elemA);
+      const bToNum = toNumber(elemB);
+
+      if (Number.isNaN(aToNum)) {
+        return !isSecondClick
+          ? elemA.localeCompare(elemB)
+          : elemB.localeCompare(elemA);
+      }
+
+      return !isSecondClick ? aToNum - bToNum : bToNum - aToNum;
+    });
+
+    tableBody.innerHTML = '';
+
+    columnsSorted.forEach((el) => {
+      tableBody.appendChild(el);
+    });
+  }
+});
+
+tableBody.addEventListener('click', (e) => {
+  const headerChildren = [...tableBody.children];
+  const columnName = e.target.closest('tr');
+
+  headerChildren.forEach((el) => el.removeAttribute('class'));
+  columnName.classList.add('active');
+});
+
+const formContainer = document.createElement('form');
+const nameLabel = document.createElement('label');
+const nameInput = document.createElement('input');
+const posiitionLabel = document.createElement('label');
```

Comment:
There is a typo in the variable name 'posiitionLabel'. It should be 'positionLabel'.